### DISCUSSION TITLE
SNAP BACK TO COURSE HEADING

### DISCUSSION THREAD

**Discussion by polyefko** [flybywiresim/aircraft#3555] [General]
Just a quick question that I asked over a month ago and replied they are working on it.
Happens EVERY SINGLE FLIGHT...download Simbrief and start flight...during flight plane starts to break away from course and I slightly nudge the flight controller and the plane snaps back to course. Happens every 2 minutes or so.
I also find this happens when I do a wing view or outside the plane view and it immediately starts to go astray.

Anyway....are you folks still aware of this and are working on it.?  Thanks for all you do.

**Comment by ChruutvoLuzi** [comment]
sounds like input from your controller, try adding more deadzone

**Reply by polyefko** [reply]
will give that a try

**Reply by polyefko** [reply]
That helped 95% thankyou. It only drifted like twice. And that again was when I changed my views to outside the plane views.
\Thanks again

**Reply by ChruutvoLuzi** [reply]
> That helped 95% thankyou. It only drifted like twice. And that again was when I changed my views to outside the plane views.
> \Thanks again

you`re welcome :D
